operator/src/main/java/oracle/kubernetes/operator/calls/AsyncRequestStep.java

```diff
@@ -377,14 +377,18 @@ public NextAction doPotentialRetry(Step conflictStep, Packet packet, int statusC
         }
 
         NextAction na = new NextAction();
```

Comment:
This is changed behavior and there is no corresponding unit test.

Reply:
Just to clarify - changed behavior is to fix a bug as retry logic was not working as intended. I have added unit tests multipleFailedCallbackNoRetriesLeft_verifyCompletionThrowable() and multipleFailedCallbackRetriesLeft_nextStepAppliedWithValue() in AsyncRequestStepTest to cover this changed behavior.